api/src/main/java/io/opencensus/trace/propagation/PropagationComponent.java

```diff
@@ -32,6 +32,14 @@
    */
   public abstract BinaryFormat getBinaryFormat();
 
+  /**
+   * Returns the {@link TextFormat} with the provided implementations. If no implementation is
```

Comment:
Please provide a link to the documentation of B3 and get some background about it:
https://github.com/openzipkin/b3-propagation

Reply:
Done.
Besides, for `HttpPropagationUtil`, I also added a link to the CloudTraceFormat. To make the javadoc available I also make this class public. Please let me know if this should not be done.